config/src/config/consensus_config.rs

```diff
@@ -128,6 +128,7 @@ pub struct PipelineBackpressureValues {
     // If we want to dynamically increase it beyond quorum_store_poll_time,
     // we need to adjust timeouts other nodes use for the backpressured round.
     pub backpressure_proposal_delay_ms: u64,
+    pub max_txns_to_include_in_block: u64,
```

Comment:
let's call this:

max_txns_from_block_to_execute: u64

this way all "block size counters" showing the whole block is correct, and we can add additional counter for block size after this trimming.

Reply:
Ok